sample.config.yaml

```diff
@@ -9,13 +9,18 @@ command_prefix: "!"
 matrix:
   # The Matrix User ID of the bot account
   user_id: "@bot:example.com"
-  # Matrix account password
+  # Login type (can be `password` or `token`)
+  login_type: token
+  # Matrix account password (if password login)
   user_password: ""
+  # Authenticate directly to Matrix (if token login)
+  access_token: "syt_blablablabla"
   # The public URL at which the homeserver's Client-Server API can be accessed
   homeserver_url: https://example.com
   # The device ID that is a **non pre-existing** device
   # If this device ID already exists, messages will be dropped silently in
   # encrypted rooms
+  # (if using token login type, request the token with this device_id)
```

Comment:
see above. the wording should be adjusted accordingly, like "only used with password; ignored in favor of existing device id with token login". also the existing description only really fits the password login, so i think we need to reword the whole thing.